### PR TITLE
[jsk_recognition_utils] Test lookupTransformWithDuration

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

**Comment by wkentaro** [comment]
I thought testing the utility function in jsk_recognition_utils is reasonable.

**Comment by wkentaro** [comment]
depends on https://github.com/jsk-ros-pkg/jsk_common/pull/1201

**Comment by wkentaro** [comment]
Please merge.

File: .travis.rosinstall [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# - git:
-#     uri: https://github.com/jsk-ros-pkg/jsk_common.git
-#     local-name: jsk-ros-pkg/jsk_common
+- git:
```
Comment (garaemon) [inline .travis.rosinstall]: Why needed?

**Comment by wkentaro** [comment]
depends on jsk-ros-pkg/jsk_common#1201

**Comment by wkentaro** [comment]
Because this PR depends on jsk-ros-pkg/jsk_common#1201, and jsk_common is not released after the commit, since I need to install jsk_common from source.

**Comment by wkentaro** [comment]
Please merge.

**Comment by wkentaro** [comment]
See https://github.com/jsk-ros-pkg/jsk_recognition/pull/1324

**Pull request closed without merging**